src/parser/jobs/pld/index.js

```diff
@@ -0,0 +1,7 @@
+import About from './About'
+import Goring from './Goring'
+
+export default [
+	About,
+	Goring
```

Comment:
I'm gonna bitch about the fact that this is lacking trailing commas and that's all I have to add.